### PR TITLE
Some funnies

### PR DESCRIPTION
**Pull request by oq-x**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -394,6 +406,13 @@ after ("click") score++!
 print(await next score)! //6 (when you click)
 ```
 
+Additionally, the `current` keyword lets you see into the present!!
+
+```java
+const var score = 5!
+print(current score)! //5
+```
+
```
Comment (TodePond) [inline README.md:415]: Nice!

File: README.md [hunk 2]
```diff
@@ -211,6 +211,12 @@ print(1^1)! // 1
 print(2^3)! // 8
 ```
 
+You can also use the number name, for example:
+
+```java
+print(one+two)! //3
+```
+
```
Comment (TodePond) [inline README.md:219]: Sure!

File: README.md [hunk 3]
```diff
@@ -260,6 +266,12 @@ If you want to be much less precise, you can use `=`.
 3 = 3.14! //true
 ```
 
+You can also be a lot less precise by using `||`
+
+```java
+3 || 2! //true
+```
+
```
Comment (TodePond) [inline README.md:274]: I'd drop this one, but keep the other two. Then I'd be happy to merge :)

Comment (oq-x) [reply README.md:274]: Ok

**Pull request merged**